bin/lib/nightly_versions.py

```diff
@@ -0,0 +1,87 @@
+from lib.amazon import dynamodb_client
+from lib.amazon_properties import get_properties_compilers_and_libraries
+
+
+class NightlyVersions:
+    version_table_name: str = "nightly-version"
+    exe_table_name: str = "nightly-exe"
+
+    def __init__(self, logger):
+        self.logger = logger
+        [self.cpp, _] = get_properties_compilers_and_libraries("c++", logger)
+        [self.c, _] = get_properties_compilers_and_libraries("c", logger)
+        [self.fortran, _] = get_properties_compilers_and_libraries("fortran", logger)
+
+    def asCCompiler(self, exe: str):
```

Comment:
Can we stick to the same naming convention as everywhere else please? Python uses `snake_case` for functions and `CamelCase` for classes. I don't prefer one over the other, but the mixture makes it hard for me to read. Thanks!